### PR TITLE
Added YAMLs for Ubuntu and Debian scylla-manager

### PR DESCRIPTION
**Pull request by yarongilor**
changed parameters of:
ami_id_monitor and ami_monitor_user.

### CODE REVIEW

**Comment by roydahan** [comment]
@yarongilor please fix

**Comment by yarongilor** [comment]
fixed in:
* 335bec0 - (HEAD -> ubuntu_and_debian_manager_yamls, origin/ubuntu_and_debian_manager_yamls) Added parameters of: AMI_ID_MONITOR, AMI_MONITOR_USER

File: tests/manager-regression-multiDC-set-distro.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+stress_cmd: "cassandra-stress write cl=QUORUM n=1200300 -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=2,us-west-2scylla_node_west=1)' -port jmx=6868 -mode cql3 native -rate threads=200 -pop seq=400200300..600200300"
+n_db_nodes: '2 1'
+n_loaders: 1
+n_monitor_nodes: 1
+monitor_branch: 'master'
+user_prefix: manager-regression-VERSION
```
Comment (roydahan) [inline tests/manager-regression-multiDC-set-distro.yaml]: manager-regression-DISTRO-VERSION

File: tests/manager-regression-multiDC-set-distro.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        scylla_repo: https://s3.amazonaws.com/downloads.scylladb.com/rpm/unstable/centos/branch-1.7/37/scylla.repo
+        #us_east_1:
+        #  gce_datacenter: 'us-east1-b'
+        multi_dcs:
+          gce_datacenter: 'us-east1-b us-west1-b us-east4-b'
+
+databases: !mux
+    scylla:
+        db_type: scylla
+        instance_type_db: 'i3.xlarge'
```
Comment (roydahan) [inline tests/manager-regression-multiDC-set-distro.yaml]: change to i3.large

File: tests/manager-regression-multiDC-set-distro.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            subnet_id: 'subnet-ec4a72c4 subnet-5207ee37'
+            ami_id_db_scylla: 'AMI_ID_EAST AMI_ID_WEST'
+            ami_db_scylla_user: 'centos'
+            ami_id_loader: 'AMI_ID_EAST'
+            ami_loader_user: 'centos'
+            ami_id_db_cassandra: 'ami-ada2b6c4'
+            ami_db_cassandra_user: 'ubuntu'
+            ami_id_monitor: AMI_ID_MONITOR # Debian-8: ami-cb4b94dd # Ubuntu16.4: 'ami-1c5cc366' # CentOs-7: ami-9887c6e7
+            ami_monitor_user: AMI_MONITOR_USER # admin / ubuntu / centos
+    gce: !mux
```
Comment (roydahan) [inline tests/manager-regression-multiDC-set-distro.yaml]: remove GCE

File: tests/manager-regression-multiDC-set-distro.yaml (hunk 1 above)
Comment (yarongilor) [reply tests/manager-regression-multiDC-set-distro.yaml]: done

File: tests/manager-regression-multiDC-set-distro.yaml (hunk 2 above)
Comment (yarongilor) [reply tests/manager-regression-multiDC-set-distro.yaml]: done

File: tests/manager-regression-multiDC-set-distro.yaml (hunk 3 above)
Comment (yarongilor) [reply tests/manager-regression-multiDC-set-distro.yaml]: done

**Comment by roydahan** [comment]
Backported to manager-v1.3

**Pull request merged**